nengo/connection.py

```diff
@@ -91,9 +98,9 @@ def check_transform(self, conn, rule):
 
         if rule.modifies == "weights":
             # If the rule modifies 'weights', then it must have full weights
-            if conn.is_decoded:
+            if not conn._to_neurons:
                 raise ValidationError(
-                    "Learning rule '%s' can not be applied to decoded "
+                    "Learning rule '%s' cannot be applied to decoded "
```

Comment:
I'll get rid of the phrase "decoded connections" here because it's no longer accurate.